src/templates/atwho_display.html

```diff
@@ -0,0 +1,7 @@
+<li>
+  <i class="<%= icon %>" /> <%= name %>
+    <% if (displayLabel) { %>
```

Comment:
love this!

On Tue, Oct 6, 2015 at 2:03 PM, Jonathan Emord <notifications@github.com>
wrote:

> In src/templates/atwho_display.html
> <https://github.com/dimagi/Vellum/pull/549#discussion_r41299587>:
>
> > @@ -0,0 +1,7 @@
> > +<li>
> > +  <i class="<%= icon %>" /> <%= name %>
> > +    <% if (displayLabel) { %>
>
> Hidden values won't have displayLabel
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/Vellum/pull/549/files#r41299587>.
>